src/lib/parsers/htmlHelper.ts

```diff
@@ -1,17 +1,23 @@
 import { Receiver } from '../../models/receiver';
 import { PegReceivedData } from '../../models/peg-received-data';
 
-function generateTable(receivers: Receiver[]) : string {
+function generateTable(receivers: Receiver[], section: string = null) : string {
 	let htmlTable =
 '					<table class="table pb-3">';
+	if(section) {
+		htmlTable =
+`					<table id="section-${section}" class="table pb-3 collapse">`;
+	}
+	let subsection = 0;
 
 	receivers.forEach((result: Receiver) => {
```

Comment:
forEach's callback function can take a second argument index—in this case, subsection—to allow you to get rid of the hand-touched subsection counting.

Reply:
I didn't know that was a thing, thanks! Updated PR.